.github/workflows/docs-preview-deploy.yml

```diff
@@ -47,10 +45,10 @@ jobs:
           head_sha: ${{ github.event.workflow_run.head_sha }}
           pull_requests: ${{ tojson(github.event.workflow_run.pull_requests) }}
         run: |
-          PR_NUMBER=$(jq -r '[.[] | select(.head.sha == "${{ env.head_sha }}")][0].number' <<< '${{ env.pull_requests }}')
+          PR_NUMBER=$(jq -r '[.[] | select(.head.sha == "${head_sha}")][0].number' <<< '${pull_requests}')
           {
             echo "PR_NUMBER=${PR_NUMBER}"
-            echo 'PR_HEADSHA=${{ env.head_sha }}'
+            echo 'PR_HEADSHA=${head_sha}'
```

Comment:
Likewise, the content is single quote wrapped, this would not interpolate and instead the value of `PR_HEADSHA` as an ENV will now be the string `${head_sha}` instead of the actual fixed string of a the head SHA.